hedera-mirror-rest/__tests__/schedules.test.js

```diff
@@ -175,6 +175,87 @@ const verifyExtractSqlFromScheduleFilters = (filters, expectedQuery, expectedPar
   expect(limit).toStrictEqual(expectedLimit);
 };
 
+describe('getScheduleCacheControlHeader', () => {
+  const LONGER_SCHEDULE_CACHE_CONTROL_HEADER = {'cache-control': 'public, max-age=3600'};
+  const testSpecs = [
+    {
+      name: 'Schedule has executed',
+      input: {
+        executed_timestamp: '1234567890000000002',
```

Comment:
this is still wrong. it should be a BigInt. some columns can be null so `null` is also appropriate depending on the test case. when it's a string, the arithmetic operations in the function like `+` will again append or prepend a number to the string.
```suggestion
        executed_timestamp: 1234567890000000002n,
```

Reply:
Updated